src/Packetery/Module/Order/Repository.php

```diff
@@ -295,15 +301,25 @@ public function getDataById( int $id ): ?object {
 	}
 
 	/**
-	 * Gets order by wc order.
-	 *
-	 * @param WC_Order $wcOrder                         WC Order.
-	 * @param bool     $suppressInvalidCarrierException Tells if carrier exception should be ignored.
-	 *
-	 * @return Order|null
-	 * @throws InvalidCarrierException InvalidCarrierException.
+	 * @throws InvalidCarrierException
 	 */
-	public function getByWcOrder( WC_Order $wcOrder, bool $suppressInvalidCarrierException = false ): ?Order {
+	public function getByWcOrder( WC_Order $wcOrder ): ?Order {
+		if ( ! $wcOrder->has_shipping_method( ShippingMethod::PACKETERY_METHOD_ID ) ) {
```

Comment:
jsou tu podle mě zbytečně duplicitní řádky jako u getByWcOrderWithValidCarrier... Oba dva ify dělají to samé. Takže  vytvořil bych si něco jako getsWcOrderDataWithPacketeryShippingMethod nebo tak něco